### DISCUSSION TITLE
Why there's no new releases?

### DISCUSSION THREAD

**Discussion by FionViber** [DevToys-app/DevToys#778] [Q&A]
People keep updating this program, but the release version stays in Sept 2022. Could you please release a new version?

**Comment by veler** [comment]
Hello,
We're actively working on DevToys 2.0, which we're revising from scratch. It takes time. We won't release major in DevToys 1.0 I'm afraid. You can check our progress on Milestone 2.0 and/or by looking at the branch 'dev/2.0'.

**Reply by veler** [reply]
See https://github.com/veler/DevToys/milestone/2
And https://github.com/veler/DevToys/tree/dev/2.0
